manifests/params.pp

```diff
@@ -21,6 +21,7 @@
   $install_grub                = false
   $remove_grub_legacy          = false
   $package_ensure              = 'present'
+  $manage_config_file          = false
```

Comment:
Should be true by default, to ensure the default behaviour does not change.

Reply:
updated to true